### PR TITLE
Reduce test output for AstAnalysisTests

### PR DESCRIPTION
**Pull request by huguesv**
Also some test fixes

CodeFlow may be better at showing the actual changes - most lines just have increased indentation

Edit: I can confirm that CodeFlow is much better for this one

### CODE REVIEW

File: Python/Tests/Analysis/AstAnalysisTests.cs [hunk 1]
```diff
@@ -399,37 +445,43 @@ public class AstAnalysisTests {
...
-                foreach (var err in errors) {
-                    Console.WriteLine(err);
-                }
-                Assert.AreEqual(0, errors.Count(), "Parse errors occurred");
+                try {
+                    var fact = (AstPythonInterpreterFactory)analysis.Analyzer.InterpreterFactory;
+                    var interp = (AstPythonInterpreter)analysis.Analyzer.Interpreter;
+
+                    var mod = interp.ImportModule(interp.BuiltinModuleName);
+                    // TODO: CreateAnalysis specifies UseExistingCache = false,
```
Comment (huguesv) [inline Python/Tests/Analysis/AstAnalysisTests.cs]: If printing this is important, I can save it and print it out in cleanup if test fails. If not important, I can delete the code.

Comment (zooba) [reply Python/Tests/Analysis/AstAnalysisTests.cs]: Yeah, we want this if the test fails.

File: Python/Tests/Analysis/AstAnalysisTests.cs [hunk 2]
```diff
@@ -399,37 +445,43 @@ public class AstAnalysisTests {
...
-                foreach (BuiltinTypeId v in Enum.GetValues(typeof(BuiltinTypeId))) {
-                    var type = interp.GetBuiltinType(v);
-                    Assert.IsNotNull(type, v.ToString());
-                    Assert.IsInstanceOfType(type, typeof(AstPythonBuiltinType), $"Did not find {v}");
-                }
+                    // Ensure we can get all the builtin types
+                    foreach (BuiltinTypeId v in Enum.GetValues(typeof(BuiltinTypeId))) {
+                        var type = interp.GetBuiltinType(v);
+                        Assert.IsNotNull(type, v.ToString());
+                        Assert.IsInstanceOfType(type, typeof(AstPythonType), $"Did not find {v}");
```
Comment (zooba) [inline Python/Tests/Analysis/AstAnalysisTests.cs]: If the type is not `AstPythonBuiltinType`, the test should fail.

File: Python/Tests/Analysis/AstAnalysisTests.cs (hunk 1 above)
Comment (zooba) [reply Python/Tests/Analysis/AstAnalysisTests.cs]: The `ImportModule` call above generates this file.

**Pull request merged**